.github/workflows/build.yml

```diff
@@ -25,20 +26,34 @@ jobs:
           - os: ubuntu-22.04
             # use old linux so that the shared library versioning is more portable
             artifact_name: capa
-            asset_name: linux
+            asset_name: linux-amd64
+            python_version: '3.10'
+          - os: ubuntu-22.04-arm
+            artifact_name: capa
+            asset_name: linux-arm64
             python_version: '3.10'
           - os: ubuntu-22.04
             artifact_name: capa
             asset_name: linux-py312
```

Comment:
```suggestion
            asset_name: linux-amd64-py312
```

Reply:
No longer relevant due to https://github.com/mandiant/capa/pull/2691#discussion_r2138740671